frontend/src/admin/components/dropdown-menu/dropdown-menu.tsx

```diff
@@ -0,0 +1,41 @@
+import React, {useState} from "react";
+import DropdownMenuIcon from "../dropdown-menu-icon/dropdown-menu-icon";
+import "./dropdown-menu.scss";
+
+interface DropdownMenuProps{
+    items: string[]
+    onClick: (value: string) => void
+}
+
+export default function DropdownMenu( {items, onClick}: DropdownMenuProps) {
+
+    const [showDropdown, setShowDropdown] = useState<boolean>(false);
+
+    const toggleDropdown = () => {
+        setShowDropdown(!showDropdown);
+    };
```

Comment:
Any specific reason this is defined as a constant variable rather than a function? Same goes for the `hideDropdown` down below.

Reply:
Nope, this changed to functions